src/Build/BackEnd/Components/RequestBuilder/RequestBuilder.cs

```diff
@@ -699,27 +694,9 @@ private async Task RequestThreadProc(bool setThreadParameters)
                 {
                     SetCommonWorkerThreadParameters();
                 }
-#if (!STANDALONEBUILD)
-                using (new CodeMarkerStartEnd(CodeMarkerEvent.perfMSBuildEngineBuildProjectBegin, CodeMarkerEvent.perfMSBuildEngineBuildProjectEnd))
-                {
-#if MSBUILDENABLEVSPROFILING
-                try
-                {
-                    string beginProjectBuild = String.Format(CultureInfo.CurrentCulture, "Build Project {0} - Start", requestEntry.RequestConfiguration.ProjectFullPath);
-                    DataCollection.CommentMarkProfile(8802, beginProjectBuild);
-#endif
-#endif
+                MSBuildEventSource.Log.RequestThreadProcStart();
```

Comment:
Path was dropped -- intentionally?

Reply:
This information is not captured in BuildProject. The path used in CodeMarkers does not seem to exist at this point.